src/domain/game.js

```diff
@@ -7,24 +7,31 @@ export const initialState = () => {
   const board = rows.map(() => new Array(BOARD_SIZE).fill(''));
   const [firstSymbol] = symbols;
   return {
-    players: [],
```

Comment:
players as an array is not a good model for this game ... as we have only 2 players X and O (Make the impossible state impossible)